python/py-pybonjour/Portfile

```diff
@@ -12,12 +12,9 @@ description         pybonjour provides a pure-Python interface to Apple \
                     Bonjour and compatible DNS-SD libraries (such as Avahi).
 long_description    ${description}
 
-homepage            https://code.google.com/p/pybonjour/
-master_sites        googlecode:pybonjour
-distname            pybonjour-${version}
+fetch.type          git
+git.url             https://github.com/DynamicDevices/pybonjour-python3.git
 
-checksums           md5     30cbfd3e9e9721b39f6aa67df1c315a2 \
-                    sha1    92cabd14e04c5f62ce067c47c2057ee3d424d29b \
-                    rmd160  87b8705f47c2b11488fe6bdcef1158da152c7a43
+depends_fetch       port:git
 
-python.versions     27
+python.versions     27 34 35 36 37 38 39 310
```

Comment:
the name of the repository already suggests that PY27 is not supported, so that version should be removed here - provided that there are no dependents. Additionally, we do not want to add EOL Python versions so is there are no other ports using this then only add PY310 and perhaps still PY39, nothing else